pkg/anchor/docker.go

```diff
@@ -8,107 +8,43 @@ import (
 
 	"github.com/fatih/color"
 	"github.com/google/go-containerregistry/pkg/crane"
-	"github.com/moby/buildkit/frontend/dockerfile/parser"
 )
 
-func attachDockerSha(node *parser.Node) (string, error) {
-	if node.Next != nil && strings.ToLower(node.Next.Value) == "as" {
-		color.Blue("Parsing %s image...", node.Next.Next.Value)
-	} else {
-		color.Blue("Parsing the final image...")
+func processFromCommand(node *Node) (string, error) {
+	if node.CommandType != CommandFrom {
+		return "", fmt.Errorf("Node is not a FROM command")
```

Comment:
Shouldn't error string not be capitalised? There are a few other instances of this with `Errorf`s

Reply:
Yeah I'll update this